### PR TITLE
Update TD Waterhouse/Direct Investing

### PR DESCRIPTION
**Pull request by ghost**
updated td Waterhouse 2fa

### CODE REVIEW

**Review by ghost — COMMENTED** [review]
added doc

**Review by Carlgo11 — DISMISSED** [review]
It looks like you're trying to add a branch of TD Canada called TD Direct Investing. Is this correct?
It also looks like TD Canada have separate logins for banking and investing.

I suggest you add TD Direct Investing as a separate entity instead of modifying the existing TD Canada entity.

File: _data/investing.yml [hunk 1]
```diff
@@ -172,9 +172,12 @@ websites:
 
     - name: TD Waterhouse
```
Comment (Carlgo11) [inline _data/investing.yml:173]: Looks like they're not using "Waterhouse" in their name any longer.
```suggestion
    - name: TD Direct Investing
```

File: _data/investing.yml [hunk 2]
```diff
@@ -172,9 +172,12 @@ websites:
...
       url: https://www.tdwaterhouse.ca
```
Comment (Carlgo11) [inline _data/investing.yml:174]:
```suggestion
       url: https://tddirectinvesting.ca
```

File: _data/investing.yml [hunk 3]
```diff
@@ -172,9 +172,12 @@ websites:
...
-      twitter: TD_Canada
+      twitter: TD_DirectInvest
```
Comment (Carlgo11) [inline _data/investing.yml:175]: Twitter handles aren't used when the site supports 2FA.
```suggestion
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
Just a few things that needs to be changed.

**Pull request closed without merging**